docs/upgrading/4_10_upgrade_other.md

```diff
@@ -1,9 +1,9 @@
-# 4.10 Other code upgrades
+# 4.9 Other code upgrades
```

Comment:
Change file name as well?
Also, please remember to add redirections for this and the old 4.9 page after merging.